src/main/java/uk/gov/register/views/EntryProof.java

```diff
@@ -31,7 +31,7 @@ public String getEntryNumber() {
 
     @SuppressWarnings("unused, used as jsonproperty")
     @JsonProperty("merkle-audit-path")
-    public List<String> getAuditPath() {
+    public List<HashValue> getAuditPath() {
```

Comment:
This method needs to be annotated with `@JsonSerialize(using = ToStringSerializer.class)`. This will correct the problem where the endpoint is currently returning `"merkle-audit-path":[{"value":"cc3c7f8f149fb415152a46d0c35098bba6644271402e8c68138e8b0328848ac0"}]`